data/tools/GUI.pyw

```diff
@@ -56,38 +56,39 @@ WESNOTH_TRAN_DIR = os.path.join(WESNOTH_ROOT_DIR, "translations")
 _ = lambda x: x
 
 
-def set_default_locale():
+def set_user_locale():
     global _
 
     # TODO: Replace CLI args for a proper locale selection GUI.
     # More importantly, code to dynamically update the text/layout is missing.
     parser = argparse.ArgumentParser(
-        description=_("Open a Graphical User Interface (GUI) to WML Tools"),
-        usage=("GUI.pyw [--lang=" + _("LANGUAGE") + "]")
+        description=_("Open a Graphical User Interface (GUI) to WML Tools")
     )
     parser.add_argument(
         '--lang',
-        # TRANSLATORS: ISO 15897 "Procedures for the registration of cultural elements" specifies use
-        # of underscores in the locale id. For example, use en_GB, not en-GB.
-        help=_("Launch GUI.pyw in the given language. Language code must follow ISO 15897 (e.g. en_GB).")
+        help=_("Launch GUI.pyw in the specified language. Language code is expected as a POSIX locale name, refer to "
+               "gettext.wesnoth.org for a full list.")
```

Comment:
Having a single translatable string split across multiple lines works, right? I'd assume it does, but figured I'd ask.

Reply:
I understand it resolves to a single line. I split text as per Python conventions, though I recognise parts of this code do not follow that convention.